Zend/tests/constexpr/new_anon_class.phpt

```diff
@@ -1,10 +1,13 @@
 --TEST--
-New with anonymous class is not supported in constant expressions
```

Comment:
Do we need a new test here for other contexts? Same goes for all tests in `Zend/tests/constexpr`.